7-animation/2-css-animations/article.md

```diff
@@ -70,25 +70,25 @@ growing.onclick = function() {
 </script>
 ```
 
-Now let's cover animation properties one by one.
+Now, let's cover animation properties one by one.
 
 ## transition-property
 
-In `transition-property` we write a list of property to animate, for instance: `left`, `margin-left`, `height`, `color`.
+In `transition-property` we write a list of properties to animate, for instance: `left`, `margin-left`, `height`, `color`.
 
-Not all properties can be animated, but [many of them](http://www.w3.org/TR/css3-transitions/#animatable-properties-). The value `all` means "animate all properties".
+Not all properties can be animated, but [many of them can](http://www.w3.org/TR/css3-transitions/#animatable-properties-). The value `all` means "animate all properties".
```

Comment:
Id in the URL, aka URI fragment, does not point anywhere in the document. I think it's better to replace W3C link with MDN. MDN page has a list of animatable properties.

```suggestion
Although not all properties can be animated, [almost all of the properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties)  generally used can be animated. The value `all` means "animate all properties".
```

"almost all of the properties generally used"  OR
"almost all of the generally used properties"  OR
"almost all of the generally used ones" ?

Reply:
Looks good.